platform/ui-next/src/components/SegmentationTable/AddSegmentRow.tsx

```diff
@@ -0,0 +1,62 @@
+import React from 'react';
+import { Button, Icons } from '@ohif/ui-next';
+import { useSegmentationTableContext } from './SegmentationTableContext';
+
+export const AddSegmentRow: React.FC<{
```

Comment:
Docs

Reply:
Really all I'm looking for is just a comment indicating  that this is used to add a segment row to the segmentation table - I know I can get most of that from the path, so perhaps a tiny bit more.